docs/src/api/http.md

```diff
@@ -156,11 +156,9 @@ health-check mechanism for use by load balancers or other network
 infrastructure. This request will always return a HTTP 200 OK response with a body of
 "ok", "behind" or "unknown" based on the following conditions:
 
-1. If one or more `--known-validator` arguments are provided to `solana-validator` - "ok" is returned
-   when the node has within `HEALTH_CHECK_SLOT_DISTANCE` slots of the highest
-   known validator, otherwise "behind". "unknown" is returned when no slot
-   information from known validators is not yet available.
-2. "ok" is always returned if no known validators are provided.
+1. If the node is within `HEALTH_CHECK_SLOT_DISTANCE` slots from the latest cluster confirmed slot, "ok" is returned.
```

Comment:
Suggestion: prefix the lines with the return code.

- `ok`: The node is within `HEALTH_CHECK_SLOT_DISTANCE` slots from the latest cluster confirmed slot.
- `behind { distance }`: The node is behind more than `distance` slots from the latest cluster confirmed slot where `distance` is greater than `HEALTH_CHECK_SLOT_DISTANCE`.
- `unknown`: The node is unable to determine its' slot distance from the cluster.

Reply:
I like this, done in https://github.com/solana-labs/solana/pull/33651/commits/fc3649e0f2168a36f1f206b422de6222372532f5